tensorflow/examples/udacity/5_word2vec.ipynb

```diff
@@ -422,38 +422,44 @@
         "graph = tf.Graph()\n",
         "\n",
         "with graph.as_default():\n",
```

Comment:
Thanks for doing this. May I ask you to do it on one line instead:
  with graph.as_default(), tf.device('/cpu:0'):
that will remove all the space changes.